internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -92,7 +92,7 @@ var (
 				Cilium:  &AddonVersion{"1.5.3", 0},
 				Kured:   &AddonVersion{"1.2.0", 0},
 				Dex:     &AddonVersion{"2.16.0", 1},
-				Gangway: &AddonVersion{"3.1.0", 1},
+				Gangway: &AddonVersion{"3.2.0", 1},
```

Comment:
Please, bump the manifest version as well here... this number should be always bumped in any case.

Reply:
Hmmm... I thought we need to bump version when skuba has a new version release. Am I right?

Reply:
Yes, in that case it's better if we both bump the version and the manifest version. As per [the RFC](https://github.com/SUSE/caasp-rfc/blob/master/accepted/0043-addon-upgrade.md), the logic could only check the manifest version (despite the current addon logic is actually taking into account both version and manifest version)